plasma_framework/contracts/src/exits/payment/PaymentInFlightExitModelLib.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity ^0.5.0;
+
+import "../../utils/Bits.sol";
+import { PaymentExitDataModel as ExitModel } from "./PaymentExitDataModel.sol";
+
+library PaymentInFlightExitModelLib {
+    using Bits for uint256;
+
+    uint8 constant public MAX_INPUT_NUM = 4;
+    uint8 constant public MAX_OUTPUT_NUM = 4;
+
+    function setInputPiggybacked(ExitModel.InFlightExit storage ife, uint16 index)
+        internal
+    {
+        ife.exitMap = ife.exitMap.setBit(uint8(index));
```

Comment:
`uint8(index)` it potentially loose precision. Are we sure values here will fit 8-bit only?

Reply:
`setBit` itself have 2^8 (256) bits to be set/flag actually. And the index of PaymentTx is bounded by `MAX_INPUT_SIZE` (4 for now) so even though I have a preference of uint16 to represent index in general, it is more than safe to claim it enough for PaymentTx. However, as the `setBit` only handles 2^8 bits, it's capped to uint8 when choosing the position to flag.

Aw, I personally treat storage as optimization so sometimes we store things with smaller size. But I just feel being consistent in other places might be easier?